services/issue/label.go

```diff
@@ -74,7 +26,7 @@ func AddLabel(issue *models.Issue, doer *models.User, label *models.Label) error
 		return err
 	}
 
-	sendLabelUpdatedWebhook(issue, doer)
+	notification.NotifyIssueChangeLabels(doer, issue, nil, nil)
```

Comment:
```suggestion
	notification.NotifyIssueChangeLabels(doer, issue, labels, nil)
```
why not use the new parameter - we can implement the handling at NotifyIssueChangeLabels later?